sdk/storage/azure-storage-blob/azure/storage/blob/_blob_utils.py

```diff
@@ -42,6 +44,7 @@
 
 _LARGE_BLOB_UPLOAD_MAX_READ_BUFFER_SIZE = 4 * 1024 * 1024
 _ERROR_VALUE_SHOULD_BE_SEEKABLE_STREAM = '{0} should be a seekable file-like/io.IOBase type stream object.'
+LeaseClient = TypeVar("LeaseClient")
```

Comment:
Why create a TypeVar instead of import LeaseClient? (though only imported if TYPE_CHECKING)

Reply:
lease client and blob utils will then have circular dependency. :(